catalog-spec/catalog-spec.md

```diff
@@ -1,24 +1,28 @@
 # STAC Catalog Specification
 
-- [Catalog fields](#catalog-fields)
-  - [Additional Field Information](#additional-field-information)
-    - [stac_extensions](#stac_extensions)
-  - [Link Object](#link-object)
-    - [Relation types](#relation-types)
-- [Media Types](#media-types)
-  - [Media Type for STAC Catalogs](#media-type-for-stac-catalogs)
-  - [STAC Media Types](#stac-media-types)
-- [Extensions](#extensions)
-
-This document explains the structure and content of a STAC Catalog. A STAC Catalog is a 
-[Collection](../collection-spec/collection-spec.md) of [STAC Items](../item-spec/item-spec.md). These Items can be 
-linked to directly from a Catalog, or the Catalog can link to other Catalogs (often called sub-Catalogs) that contain 
-links to Items. The division of sub-catalogs is up to the implementor, but is generally done to aid the ease of 
-online browsing by people.
-
-Catalogs are not intended to be queried. Their purpose is discovery: to be browsed by people and crawled
-by machines to build a search index. A Catalog can be represented in JSON format. Any JSON object 
-that contains all the required fields is a valid STAC Catalog.
+- [STAC Catalog Specification](#stac-catalog-specification)
```

Comment:
How useful is a link to the heading directly above? Also I don't think we do this in the other TOCs. So would remove this again.